generate-packaging-dir

```diff
@@ -1,32 +1,9 @@
 #!/bin/sh
 set -eu
 
-fetch1404() (
-    set -u
-    git fetch --quiet "https://github.com/$TRAVIS_PULL_REQUEST_SLUG" "$TRAVIS_PULL_REQUEST_BRANCH-14.04"
-)
+. /etc/os-release
 
-if [ ! -d .git ]; then
-    echo "not running in a git checkout, skipping"
-    exit 0
-fi
+# update symlink
+rm -f debian
+ln -s packaging/${ID}-${VERSION_ID} debian
```

Comment:
I will add FIXMEs - as soon as we add debian or a non-debian system we can fix this script easily.